test/e2e/launch.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
you should add `set -x` in order to print the script output

```suggestion
#!/bin/bash
set -x
```